controllers/base.go

```diff
@@ -71,6 +71,18 @@ func (c *ApiController) ReadRequestFromQueryParams() BaseDataManageRequest {
 		Organization: c.Input().Get("organization"),
 	}
 
+	if !util.FilterField(result.Field) {
```

Comment:
Name of the FilterField() seems off.
Looks like it must be IsFieldValueAllowedForDB or something like that.

Reply:
done